docs/index.rst

```diff
@@ -113,6 +113,11 @@ To enable this hook, add it as the ``tm.manager_hook`` setting in your app.
        config.include('pyramid_tm')
        # ...
 
+This specific example, using an explicit mode non-threadlocal manager, is
+highly recommended and is shipped as :func:`pyramid_tm.explicit_manager`.
+Simply set ``tm.manager_hook = pyramid_tm.explicit_manager`` in your settings
```

Comment:
It will raise errors way sooner in real world projects that I have worked on, and will make sure you don't have dangling transactions hanging around.

Not sure how easy it will be to create rogue transactions with tm being over the exc views, but I've had this happen in the past and it wasn't immediately obvious. Setting up an explicit transaction manager helped find those bugs much easier.